lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -191,6 +192,71 @@ ConvertAtenReluOp::matchAndRewrite(AtenReluOp op, ArrayRef<Value> operands,
   }
 }
 
+template <>
+LogicalResult ConvertAtenOp<AtenMulTensorOp>::matchAndRewrite(
+    AtenMulTensorOp op, ArrayRef<Value> operands,
+    ConversionPatternRewriter &rewriter) const {
+  AtenMulTensorOp::Adaptor adaptor(operands);
+
+  Value lhs = adaptor.self();
+  auto lhsTy = lhs.getType().cast<TensorType>();
+  Value rhs = adaptor.other();
+  auto rhsTy = rhs.getType().cast<TensorType>();
+
+  if (!lhsTy || !rhsTy)
+    return op.emitError("Only Tensor types supported in TOSA");
+
+  auto lhsElemTy = lhsTy.getElementType();
+  auto rhsElemTy = rhsTy.getElementType();
+
+  if (lhsElemTy != rhsElemTy)
+    return op.emitError("Add: input datatypes mismatched");
+
+  if (lhsElemTy.isa<mlir::FloatType>()) {
+    rewriter.replaceOpWithNewOp<tosa::MulOp>(
+        op, getTypeConverter()->convertType(op.getType()), lhs, rhs,
+        0 /*shift*/);
```

Comment:
nit: in LLVM coding style, we usually do `/*shift=*/0`

Reply:
Done